data/points.go

```diff
@@ -0,0 +1,74 @@
+package data
+
+import (
+	"hash/fnv"
+	"sort"
+	"time"
+)
+
+// Point defines the values that will be written to the database
+type Point struct {
+	Name   string
+	Tags   Tags
+	Time   time.Time
+	Fields map[string]interface{}
+}
+
+func (p *Point) SeriesID() uint64 {
```

Comment:
I would rename this to something like `HashID`. `SeriesID` implies that it's something that is unique, which isn't guaranteed in this case.